scipy/interpolate/fitpack2.py

```diff
@@ -83,16 +83,16 @@ class UnivariateSpline:
     y : (N,) array_like
         1-D array of dependent input data, of the same length as `x`.
     w : (N,) array_like, optional
-        Weights for spline fitting.  Must be positive.  If `w` is None,
+        Weights for spline fitting. Must be positive. If `w` is None,
         weights are all equal. Default is None.
     bbox : (2,) array_like, optional
         2-sequence specifying the boundary of the approximation interval. If
-        `bbox` is None, ``bbox=[x[0], x[-1]]``. Default is None.
+        `bbox` is None, ``bbox = [x[0], x[-1]]``. Default is None.
```

Comment:
Previous case is correct as when you pass a parameter to a function you don't have spaces between the equal sign. Same bellow.

The case with ``k = 3`` vs ``k=3`` is more arguable I think as this could read more as *math*.

Reply:
I agree; according to your suggestion; line 100
```
 If `s` is None, ``s = len(w)`` which should be a good value if
```
has to be change, too??

Reply:
Yes. And now that I think about this more, please change back all `k = 3` to `k=3` (and alike). Let's just be consistent with the fact that this is a parameter and how it's called.